lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -184,50 +192,33 @@ def _finish_created_job(
             error=f"{exception}",
         )
 
-    def _prepare_to_run(self, params, concierge_params=None) -> PreparedJobParams:
+    def _prepare_to_run(self, params, concierge_params=None) -> JobSubmissionParameters:
         """
-        Creates a job record, grabs info about the objects,
-        checks the catalog resource requirements, and submits to condor
+        Creates a job record and creates the job submission params
         """
-        # perform sanity checks before creating job
-        self._check_ws_objects(source_objects=params.get("source_ws_objects"))
```

Comment:
This check is moved to `_check_job_arguments` which is run before the parent job is created